README.md

```diff
@@ -17,59 +17,33 @@
 [gc]: https://gitter.im/orgs/amethyst/rooms
 [di]: https://discord.gg/GnP5Whs
 
-**Warning: The book and tools repository are severely out of date at the moment. Amethyst is undergoing a lot of changes at the moment so if you are looking to use the library it would be better to just read the examples.**
+**Warning: The book and the documentation are missing content. Amethyst is undergoing a lot of changes at the moment.**
 
 This project is a *work in progress* and is very incomplete; pardon the dust!
-Read a summary of what happened this past week at [*This Week in Amethyst*][tw].
 
-[tw]: https://www.amethyst.rs/
+## Features
 
-### [Documentation][ar]
+Please visit the [features page][feat] for a list of features Amethyst provides and will provide.
 
-[ar]: https://www.amethyst.rs/doc/
+[feat]: docs/FEATURES.md
 
-## Vision
```

Comment:
Could have kept this part, its what actually made me join the project after all ;)

Mainly those 2:
-* Massively parallel architecture.
 -* [Correct entity-component-system model][em], with entities and prefabs